types/noble/index.d.ts

```diff
@@ -23,6 +24,14 @@ export declare function on(event: "scanStart", listener: () => void): events.Eve
 export declare function on(event: "scanStop", listener: () => void): events.EventEmitter;
 export declare function on(event: "discover", listener: (peripheral: Peripheral) => void): events.EventEmitter;
 
+export declare function removeListener(event: string, listener: Function): events.EventEmitter;
```

Comment:
More general overload should come last, as described at http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html.

Reply:
Also, consider not using the type Function as described at 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes.